server/cmd/mmctl/commands/channel_users_roles.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"context"
+
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
+
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/client"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var ChannelUsersRolesCmd = &cobra.Command{
+	Use:   "roles",
+	Short: "Management of channel users",
+}
+
+var ChannelUsersRolesAddCmd = &cobra.Command{
+	Use:   "add [team]:[channel]",
+	Short: "Give user(s) role(s) in a channel",
+	Long:  "Give user(s) role(s) in a channel",
+	Example: `  channel users roles add myteam:mychannel userA,userB roleA,roleB
```

Comment:
@ezekielchow Can you make it more clear as to how the command works please. On my first reading of this I thought userA was being assigned roleA and userB was being assigned roleB. It wasn't clear that userA and userB were being assigned both roles